### DISCUSSION TITLE
[SUGGESTION] move account_* tables from characters DB to auth

### DISCUSSION THREAD

**Discussion by Bogir** [azerothcore/azerothcore-wotlk#3933] [Ideas]
it seems, tables 

- characters@account_data
- characters@account_instance_times
- characters@account_tutorial 

must be in "auth" DB

**Comment by FrancescoBorzi** [comment]
Imagine you have a multi-realm server. This is the typical scenario:

- you have only one `auth` database, shared across all realms
- every realm has a separate `characters` and `world` databases

so when the same account can have different "account data" in different realms, we hold this data in `characters`.

If we put everything in `auth`, then in a multi-realm server this can no longer work.
